pkgs/top-level/all-packages.nix

```diff
@@ -326,6 +326,12 @@ with pkgs;
 
   banana-accounting = callPackage ../applications/office/banana-accounting { };
 
+  basalt-monado = callPackage ../by-name/ba/basalt-monado/package.nix {
+    tbb = tbb_2021_11;
+    cereal = cereal_1_3_2;
+    opencv = opencv.override{ enableGtk3 = true; };
```

Comment:
I'm not certain, but I don't think this follows https://search.nixos.org/packages?show=nixfmt-rfc-style

Reply:
```suggestion
    opencv = opencv.override { enableGtk3 = true; };
```